app/core/tests/test_api.py

```diff
@@ -169,6 +169,13 @@ def test_create_event(auth_client, project):
         "video_conference_url": "https://zoom.com/link",
         "additional_info": "Test description",
         "project": project.uuid,
+        "must_attend": (
+            '[{"practice_area":"Professional Development","permission_type":"adminProject"},'
+            '{"practice_area":"Development","permission_type":"practiceLeadProject"},'
+            '{"practice_area":"Design","permission_type":"practiceLeadJrProject"}]'
+        ),
+        "should_attend": '[{"practice_area:"Development","permission_type":"memberProject"}]',
+        "could_attend": '[{"practice_area:"Design","permission_type":"memberGeneral"}]',
```

Comment:
I missed this before. This should be plain JSON as well. So just remove the parentheses and single quotes from this block.

In the `should_attend` and `could_attend` lines, `practice_area` is missing the closing quote